pages/common/perl.md

```diff
@@ -21,3 +21,11 @@
 - Run a Perl script in debug mode, using `perldebug`:
 
 `perl -d {{script.pl}}`
+
+- Loo[p] over all lines of a file, editing them [i]n-place using a find/replace [e]xpression:
+
+`perl -p -i -e 's/{{find}}/{{replace}}/g' {{filename}}`
+
+- Replace all occurrences of a string in a file with another string, saving the original file with a given extension:
+
+`perl -p -i'.old' -e 's/{{find}}/{{replace}}/g' {{filename}}`
```

Comment:
Not sure you mean second of the two alternatives I presented, or second among the three descriptions I listed. I'll assume the former, but feel free to correct me afterwards :)